project.godot

```diff
@@ -453,6 +453,7 @@ g_interact={
 
 3d_physics/layer_1="Common"
 3d_physics/layer_2="Microbe"
+3d_physics/layer_3="Misc"
```

Comment:
Can't the layer be 32? It'd be better to leave a bunch of empty spaces for other more useful stuff to be earlier.

Reply:
Hold on, that seem to mess with the raycast collision mask.

Reply:
I was just about to comment on that. Maybe layer 32 is a special debug layer, and our last real layer should be 31?
I see in the raycast that you copied the Godot default values for the mask, that might be what excludes the last layers.

Reply:
Probably best to use the last layer that Godot allows to be detected by raycasts by default so that there's no surprising bug when someone uses a mix of our raycast and Godot raycast method, which could happen if we have a different default mask.

Reply:
`AgentProjectile` need layers set in 31 and 32 then?

Reply:
Okay, I think I get it. Layer 31 is now reserved for `Misc` instead.